src/main/kotlin/services/PersonService.kt

```diff
@@ -22,6 +23,16 @@ class PersonService(
             else -> null
         }
 
+        val lastActiveAt = when {
```

Comment:
Deze logica begrijp ik niet helemaal.  Is het nog voldoende om alleen naar de form waarde te kijken ?
```kotlin
val lastActiveAt = 
  if (it.active) ZonedDateTime.now() 
  else this.lastActiveAt
```

Reply:
Nee, je wilt het moment opslaan dat de Person van active naar inactive gaat, dat is wanneer de form status inactive is, maar de opgeslagen entity is nog active.